release/src/main/scripts/cut_release_branch.sh

```diff
@@ -127,10 +124,6 @@ echo "==================Current working branch======================="
 echo ${RELEASE_BRANCH}
 echo "==============================================================="
 
-sed -i -e "s/${DEV}/${RELEASE}/g" gradle.properties
-sed -i -e "s/${DEV}/${RELEASE}/g" sdks/python/apache_beam/version.py
-sed -i -e "s/${DEV}/${RELEASE}/g" sdks/go/pkg/beam/core/core.go
-# TODO: [BEAM-4767]
 sed -i -e "s/'beam-master-.*'/'beam-${RELEASE}'/g" runners/google-cloud-dataflow-java/build.gradle
```

Comment:
Retroactive CC @emilymye that this is the only remaining edit of the release branch at cut time. I would like to also eliminate this one so the release branch can be perfectly analogous to master branch.